src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -316,7 +312,7 @@ class NetworkController(settings: NetworkSettings,
 
     val handler = context.actorOf(handlerProps) // launch connection handler
     context.watch(handler)
-    val connectedPeer = ConnectedPeer(connectionId, handler, None)
+    val connectedPeer = ConnectedPeer(connectionId, handler, 0, None)
```

Comment:
Why None? Maybe possible to get peer info from PeerManager (ConfirmConnection handling)

Reply:
peerInfo will be set when handshake is established.
peerInfo.notEmpty is used now to filter connected peers.
I wonder if we need to distinguish connected and handshaked peers?